addons/l10n_ae/views/account_move.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
+        <record id="view_move_form" model="ir.ui.view">
+            <field name="name">View Move Form l10n_AE</field>
```

Comment:
check how we name in other localizations.

Reply:
Fixed